src/peergos/server/Main.java

```diff
@@ -540,6 +540,9 @@ public static UserService startPeergos(Args a) {
 
             BatCave batStore = new JdbcBatCave(getDBConnector(a, "bat-store", dbConnectionPool), sqlCommands);
             BlockRequestAuthoriser blockRequestAuthoriser = Builder.blockAuthoriser(a, batStore, hasher);
+            if (useIPFS) {
+                ipfsWrapper.setBlockRequestAuthoriser(blockRequestAuthoriser);
```

Comment:
what happens to reqs it receives before this is set?

Reply:
It's still a race condition as ipfs is started and listening by then. What happens to those requests?